app/jobs/find_available_user_barcode_job.rb

```diff
@@ -46,7 +47,7 @@ def perform(user: nil, **args)
       # well, we can't be saving this user with a duplicate barcode.
       # ask the user to increment its barcode, and try again.
       if already_there
```

Comment:
`already_there` is not a clear variable name for someone new to the code. Does this refer to the barcode? Is it "there" in Sierra or in MLN?

Reply:
fair point.  the variable is set in the base branch, I'll update it its own PR.